djimaging/tables/response/orientation.py

```diff
@@ -220,18 +245,18 @@ def compute_os_ds_idxs(snippets: np.ndarray, dir_order: np.ndarray, dt: float):
     osi, pref_or = get_si(dir_component, sorted_directions, 2)
     (t, d, r) = sorted_responses.shape
     temp = np.reshape(sorted_responses, (t, d * r))
-    projected = np.dot(np.transpose(temp), time_component)  # we do this whole projection thing to make the result
-    projected = np.reshape(projected, (d, r))  # between the original and the shuffled comparable
+    projected_flat = temp.T @ time_component  # we do this whole projection thing to make the result
+    projected = np.reshape(projected_flat, (d, r))  # between the original and the shuffled comparable
```

Comment:
Functionally the same as before, just maybe a bit more readable (personally I feel np.dot should be used when it is clearly just a vector dot product of 1D vectors, otherwise matmul/@ is clearer).

Reply:
ok